hugo-extended.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: hugo-extended
+  version: 0.112.3
+  epoch: 0
+  description: The world's fastest framework for building websites.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    # required for extended SASS libraries
+    # - libc6-compat
+    - libstdc++
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - git
+
+pipeline:
+  - uses: go/install
+    with:
+      package: github.com/gohugoio/hugo
+      version: v${{package.version}}
+      tags: extended
```

Comment:
Is it possible to do the extended build in the same pipeline as `hugo.yaml`? Is it appropriate to use subpackages in this case?

Reply:
It could be a sub-package, since it's more of a build tag than a related package that's self-contained I'd say keep as two-separate packages. I noticed other Linux distros had one monolithic package for extended only, but I think two separate packages makes most sense here